src/containers/facility/FacilityOffers.tsx

```diff
@@ -1,15 +1,60 @@
 import { useParams } from 'react-router-dom';
 import { RoomCard } from 'src/components/RoomCard';
 import { useAccommodationsAndOffers } from 'src/hooks/useAccommodationsAndOffers.tsx';
+import { styled, Typography, useTheme } from '@mui/material';
+import { Box } from '@mui/material';
+import { forwardRef } from 'react';
 
-export const FacilityOffers = () => {
+interface SearchCriteriaAndResult {
+  rooms?: number;
+  guests: number;
+  startDate: string;
+  nights: number;
+  roomsAvailable?: number;
+}
+
+const FacilityOffersContainer = styled(Box)(({theme}) => ({
+  marginBottom: theme.spacing(2.5),
+  padding: theme.spacing(2.5)
+}));
+
+const FacilityOffersTitle = ({
+  rooms,
+  roomsAvailable,
+  guests,
+  startDate,
+  nights
+}: SearchCriteriaAndResult) => {
+
+  const SubHeader = styled(Box)(() => ({}));
+  const theme = useTheme();
+
+  return (
+    <Box marginBottom={theme.spacing(2.5)}>
+      <Typography marginBottom={theme.spacing(6)}>Available Rooms</Typography>
```

Comment:
it also possible to use `mb` instead of `marginBottom`. no need to change, just sharing knowledge.